hw_8_new.py

```diff
@@ -0,0 +1,57 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+import time
+def my_decorator(func):
+    def deco_func():
```

Comment:
Use *args and **kwargs here to make in works with functions that take arguments. And pass these arguments to the func() call